synapse/storage/databases/main/roommember.py

```diff
@@ -1368,6 +1368,55 @@ def _is_local_host_in_room_ignoring_users_txn(
             _is_local_host_in_room_ignoring_users_txn,
         )
 
+    async def forget(self, user_id: str, room_id: str) -> None:
+        """Indicate that user_id wishes to discard history for room_id."""
+
+        def f(txn: LoggingTransaction) -> None:
+            sql = (
+                "UPDATE"
+                "  room_memberships"
+                " SET"
+                "  forgotten = 1"
+                " WHERE"
+                "  user_id = ?"
+                " AND"
+                "  room_id = ?"
+            )
+            txn.execute(sql, (user_id, room_id))
```

Comment:
While we're here can we change this to a `simple_update` please? The query hurts my eyes.